Examples/zib-Pregnancy-Gravidity - example.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="http://hl7.org/fhir/STU3/observation.sch" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<Observation xmlns="http://hl7.org/fhir" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://hl7.org/fhir http://hl7.org/fhir/STU3/observation.xsd">
+    <id value="zib-pregnancy-gravidity-example"/>
+    <meta>
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/zib-Pregnancy-Gravidity"/>
+    </meta>
+    <extension url="http://hl7.org/fhir/StructureDefinition/event-partOf">
+        <valueReference>
+            <reference value="Observation/zib-pregnancy-example" />
+            <display value="Example pregnancy" />
+        </valueReference>
+    </extension>
+    <status value="final"/>
+    <code>
+        <coding>
+            <system value="http://loinc.org"/>
+            <code value="11996-6"/>
+            <display value="Gravidity"/>
+        </coding>
+    </code>
+    <subject>
+        <reference value="Patient/example"/>
+        <display value="Example Patient"/>
+    </subject>
+    <valueQuantity>
+        <value value="2"/>
```

Comment:
No unit?

Reply:
Correct me if I'm wrong, maar volgens mij is de conventie voor dimensieloze eenheden in UCUM om accolades te gebruiken, dit is hetzelfde als 1 maar je kan er iets tussen zetten wat voor mensen te begrijpen is. Dus in dit geval zouden we bv. '{times}' kunnen schrijven.

Reply:
Hoi Pieter, UCUM zegt dit: "Users are free to use curly braces expressions if they think it is important to use symbols rather than the default unit 1.  Curly braces expressions are equivalent to the unit 1. The details of the annotations in the curly braces have no defined meaning in The Unified Code for Units of Measure."

En FHIR zegt over datatype "Count" (variation on Quantity): "There SHALL be a code with a value of "1" if there is a value. If system is present, it SHALL be UCUM. If present, the value SHALL be a whole number." Graviditeit en pariteit zijn uiteraard "count".

Dus UCUM staat "1" toe - naast accolades - en FHIR wil "1", geen accolades. Eens? Anders moeten we Alexander even vragen hoe dit elders gedaan wordt.

Reply:
Hoi Marc, ik kende Count nog niet als sub-datatype, maar zie ook gelijk dat het niet inzetbaar is in Observation (het wordt nergens gebruikt op het moment). Daarmee hebben we er volgens mij ook niet zo veel aan, we kunnen de validatiekracht iig niet uitbuiten.
Daarmee worden Graviditeit en Pariteit dus gewoon Quantity's. En daarvoor zou ik toch wel willen pleiten dat '{count}' of '{times}' oid leesbaarder is dan '1'.